custom/icds_reports/static/js/directives/awc-opened-yesterday/awc-opened-yesterday.directive.js

```diff
@@ -5,7 +5,7 @@ function AwcOpenedYesterdayController($routeParams, $location, storageService, s
     var vm = this;
     vm.data = {};
     vm.step = $routeParams.step;
-    vm.filters = [];
+    vm.filters = ['data_period'];
```

Comment:
just confiming what you mentioned about this is used to exclude the filter which are not to be shown.
can you confirm here that this page is showing other filter like age and gender filters?
and it was empty before does that mean it was showing all filters?

Reply:
I dont see this particular report anywhere on dashboard